apps/examples/testcase/le_tc/filesystem/fs_main.c

```diff
@@ -707,6 +801,27 @@ static void tc_fs_vfs_rewinddir(void)
 	ret = closedir(dirp);
 	TC_ASSERT_EQ("closedir", ret, OK);
 	TC_ASSERT_EQ("rewinddir", count, VFS_LOOP_COUNT * 2);
+
+	/*For Pseudo dir operations */
+	dirp = opendir(ROOT);
+	TC_ASSERT_NEQ("opendir", dirp, NULL);
+
+	count = 0;
+	temp_count = 0;
+
+	while ((dirent = readdir(dirp)) != NULL) {
```

Comment:
Does it not required to check `dirp` against NULL before readdir(), if opendir() has failed?

Reply:
No, if opendir is failed it will return the value. So readdir will not be executed.